### ISSUE TITLE
Typo in BatchNormalization layer?

### ISSUE THREAD

**Issue by wizardfishball** [tensorflow/tfjs#43]
#### TensorFlow.js version
           "@tensorflow/tfjs-layers": "0.1.2"

#### Describe the problem or feature request
          https://github.com/tensorflow/tfjs-layers/blob/master/src/layers/normalization.ts#L228
          In the `normalizeInference()`, current code is `const broadcastGamma = this.center ? K.reshape(this.gamma.read(), broadcastShape) : null;` 
          it differs from the implementation in Keras which states `if self.scale: broadcast_gamma = K.reshape(self.gamma, broadcast_shape)`.

        I'm not entirely sure if this is a typo.
